package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "dependabot-pull-request-action",
-  "version": "1.1.1",
+  "version": "1.2.1",
```

Comment:
v1.2.0 does exist, the package.json just didn't get bumped as our versioning is done via Releases.

I added a script wrapper for `npm version` we use on another action to make this a little smoother.

Reply:
Oops that was my bad. Nice catch!

Reply:
Yeah, I only caught it as I'd left `updater-action` at `v0.0.0` through about six releases 😬